test/runtests.jl

```diff
@@ -33,6 +33,7 @@ using SafeTestsets, Test
     #if GROUP == "All" || GROUP == "Miscellaneous-NetworkAnalysis"
         # Tests various miscellaneous features.
         @time @safetestset "API" begin include("miscellaneous_tests/api.jl") end
+        @time @safetestset "Steady State Stability Computations" begin include("miscellaneous_tests/stability_computation.jl") end
         @time @safetestset "Compound Species" begin include("miscellaneous_tests/compound_macro.jl") end
         @time @safetestset "Reaction Balancing" begin include("miscellaneous_tests/reaction_balancing.jl") end
         @time @safetestset "Units" begin include("miscellaneous_tests/units.jl") end
```

Comment:
Just in terms of core functionality I would put `Units` after `API`, and probably put this stability calculation at the end or second to last. This and balancing seem the least "core".